pegasus/migrations/116_add_contact_rollups_email_malformed_column.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:contact_rollups) do
+      add_column :email_malformed, TrueClass
```

Comment:
I had to look up what TrueClass does in this [context](https://github.com/jeremyevans/sequel/blob/master/doc/schema_modification.rdoc). 
> for boolean columns, you can use either TrueClass or FalseClass, they are treated the same way (ruby doesn't have a Boolean class)